### PR TITLE
Version 1.0.6 bugfixes

### PR DESCRIPTION
**Pull request by cosminbosutar**
Bugfix: Added die('OK') at the end of the file that processes the IPN response.
Bugfix: Added the 'timestamp' to the 'identifier' when it is built for an authenticated customer.

### CODE REVIEW

File: Twispay WooCommerce Plugin/twispay/includes/twispay-processor.php [hunk 1]
```diff
@@ -94,6 +94,8 @@
                     /* , 'tags' => [] */
                     ];
 
+        error_log("customer=" . print_r($customer), true);
```
Comment (dragos-ursu) [inline Twispay WooCommerce Plugin/twispay/includes/twispay-processor.php]: Please check the line, `print_r` will output the content without the second parameter set to bool true.

**Review by dragos-ursu — CHANGES_REQUESTED** [review]
Please see the comments

File: Twispay WooCommerce Plugin/twispay/includes/twispay-processor.php (hunk 1 above)
Comment (cosminbosutar) [reply Twispay WooCommerce Plugin/twispay/includes/twispay-processor.php]: I removed the line as it was a debug leftover.

**Comment by dragos-ursu** [comment]
code review done

**Pull request merged**